src/NServiceBus.Extensions.DispatchRetries/DispatchRetriesEndpointConfigurationExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NServiceBus
+{
+    public static class DispatchRetriesEndpointConfigurationExtensions
+    {
+        private static DispatchRetriesConfiguration _config;
```

Comment:
Is it a good idea to keep a static reference to `DispatchRetriesConfiguration ` which internally has a reference to endpoint config? Why do you even need to do this?

Reply:
I'm trying to guard against something like this:

```csharp
var endpointConfig = new EndpointConfiguration("...");
endpointConfig.DisaptchRetries().DefaultImmediateDispatchRetriesPolicy(...);
endpointConfig.DisaptchRetries().DefaultBatchDispatchRetriesPolicy(...);
```

because in the `DispatchRetriesConfiguration` the feature is enabled, would it be a problem if the feature is enabled twice?

Reply:
Not at all. Features are type based as far as I remember